src/internal/tester/tconfig/config.go

```diff
@@ -45,7 +46,9 @@ const (
 	EnvCorsoM365TestSiteID           = "CORSO_M365_TEST_SITE_ID"
 	EnvCorsoM365TestSiteURL          = "CORSO_M365_TEST_SITE_URL"
 	EnvCorsoM365TestTeamID           = "CORSO_M365_TEST_TEAM_ID"
+	EnvCorsoSecondaryM365TestTeamID  = "CORSO_SECONDARY_M365_TEST_TEAM_ID"
```

Comment:
Does this need to get added to the GH env?

Reply:
I've added fallback options which should be good. But it would be good to add them to CI as well. While we are on this topic, I think we should add this new site to the list of sites to cleanup as well.